src/ast/data_type.rs

```diff
@@ -79,6 +79,16 @@ pub enum DataType {
     ///
     /// [1]: https://jakewheat.github.io/sql-overview/sql-2016-foundation-grammar.html#exact-numeric-type
     Decimal(ExactNumberInfo),
+
+    /// BigNumeric type used in BigQuery
+    ///
+    /// [bigquery]: https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#bignumeric_literals
+    BigNumeric,
+
+    /// This is alias for BigNumeric type used in BigQuery
+    ///
+    /// [bigquery]: https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#bignumeric_literals
+    BigDecimal,
```

Comment:
- Why the spaces between the data types? Seems unnecessary
- The link you put is related to nothing.
What about:
```suggestion
    /// [BigNumeric] type used in BigQuery
    ///
    /// [BigNumeric]: https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#bignumeric_literals
    BigNumeric,
    /// This is alias for [BigNumeric] type used in BigQuery
    ///
    /// [BigNumeric]: https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#bignumeric_literals
    BigDecimal,
```